paramnormal/dist.py

```diff
@@ -904,6 +904,39 @@ def fit(cls, data, **guesses):
         return cls.param_template(R=b*sigma, loc=loc, sigma=sigma)
 
 
+class truncated_normal(BaseDist_Mixin):
+    dist = stats.truncnorm
+    param_template = namedtuple('params', ['lower', 'upper', 'mu', 'sigma'])
+    name = 'truncated normal'
+
+    @staticmethod
+    @utils.greco_deco
+    def _process_args(lower=None, upper=None, mu=None, sigma=None, fit=False):
+        a = None
+        b = None
+        if lower is not None and mu is not None and sigma is not None:
+            a = (lower - mu) / sigma
+
+        if upper is not None and mu is not None and sigma is not None:
```

Comment:
Geez, that's confusing! A quick thought, likely rubbish --- how about say, fixing `upper` and `lower` from the extent of the data and then calling `truncnorm.fit_loc_scale` to estimate `mu` and `sigma`?